controls/datagrid/grouping/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+page_title: .NET MAUI DataGrid Documentation - Grouping Overview
+description: "Review the Telerik .NET MAUI DataGrid Grouping documentation article to learn more about all built in Grouping functions you can use."
+position: 0
+slug: datagrid-grouping
+---
+
+# Overview
+
+`Telerik UI for .NET MAUI DataGrid` provides a programmatic as well as a drag and drop approach for grouping its data based on a specific criteria
```

Comment:
**Telerik UI for .NET MAUI DataGrid** supports grouping operations either through the UI - using the Grouping UI or programmatically through Property and Delegate group descriptors.